modules/order/config/install/views.view.commerce_orders.yml

```diff
@@ -788,6 +788,71 @@ display:
           destination: true
           entity_type: commerce_order
           plugin_id: entity_operations
+        name:
```

Comment:
Why did you add this? We already have a Customer column that shows the user name.